packages/core/src/components/ToastContainer/ToastContainer.stories.mdx

```diff
@@ -20,7 +20,8 @@ import { PersonAddIcon } from '@medly-components/icons';
 
 # ToastContainer
 
-Raise a toast notification by triggering an action. You have to add `<ToastContainer/>` in your app at root level and then just use `addToast` function to raise the toast.
+Use the `Toast` container to raise a toast notification by calling a function. The toast closes itself when the close button is clicked, or by default, it expires after 5 seconds.
```

Comment:
Please remove the first line `Use the `Toast` container to raise a toast notification by calling a function.`